lib/AST/ProtocolConformance.cpp

```diff
@@ -1516,3 +1518,8 @@ const UnifiedStatsReporter::TraceFormatter*
 FrontendStatsTracer::getTraceFormatter<const ProtocolConformance *>() {
   return &TF;
 }
+
+void swift::simple_display(llvm::raw_ostream &out,
+                           const ProtocolConformance *conf) {
+  conf->dump(out);
```

Comment:
Should I worry about this causing other requests to fire?

Reply:
This is a placeholder... I'll fix it up.

Reply:
Oh wow, the printer can kick off interface type requests for var decls will can perform arbitrarily large amounts of type checking.  Lovely.